### ISSUE TITLE
Ability to filter by "Double Clicking" event in "Mini Chart"

### ISSUE THREAD

**Issue by l0psec** [redcanaryco/mac-monitor#13]
**At a high level -- can you summarize your request?**
Quickly filter the events provided in the main screen by double clicking the event of interest from the mini-chart. 

**What is the current alternative solution?**
Not entirely sure, if I "right click" on an event in the Event Type column for example ES_EVENT_TYPE_NOTIFY_MMAP, I can filter this event type out using "filter event", however this removes this event. What I want to do is match only this event type. 

**Anything else?**
The Mini Chart that I am most interested in being able to double click to show all those events is this:
<img width="272" alt="Screenshot 2023-04-17 at 4 15 00 PM" src="https://user-images.githubusercontent.com/67293122/232599970-552af55f-84ca-4104-b86f-0c9b4e2bc677.png">

**Comment by Brandon7CC** [comment]
Hey @l0psec! Looked into adding this a while back, but SwiftUI seemed to be a limiting factor. I'd also like to add the feature 😄 Thank you!

**Comment by Brandon7CC** [comment]
Looked into this one a bit more and while SwiftUI would allow for me to implement this feature the overlap with event masking / filtering is too great for me to get this on my backlog right now. It is something I think would be really cool to add in the future though and I'll keep a note that people are interested in expanded functionality with the mini-chart 😉
